chore: changes to be compatible with YLT v3

ohoareau opened the pull request.

ohoareau committed:
chore: changes to be compatible with YLT v3

gmetais commented:
Hi Olivier,
Happy New Year!

The "Unhandled Promise Rejection" error you encountered is related to a Puppeteer option called `headless: "new"` that was introduced in Phantomas.

The fix in Phantomas is not ready yet, so I can't update YLT, but I tested a hotfix on my local docker image. 

Here is a line I added to your Dockerfile.lambda file (this command adds a double slash at the beginning of the line 35): 

```
RUN sed -i '35 s/^/\/\//' node_modules/phantomas/lib/browser.js
```

It might be used as a temporary fix. I'll let you know when the final fix is available.

ohoareau committed:
fix: add fix for headless new

demostanis committed:
fix: remove unneeded analyzeCss.js from phantomas

this reduces the generated report data's size sent to symphony
(from ~1.1M to 300K on google.com)
